m2cgen/ast.py

```diff
@@ -49,10 +49,10 @@ def __str__(self):
 
 
 class ArrayExpr(NumExpr):
```

Comment:
Do we need to rename this to `VectorExpr`?

Reply:
Done